sdk/product-lifecycle.md

```diff
@@ -4,39 +4,49 @@ Two preliminary lifecycle stages - Alpha and Beta - are designed to ensure that
 
 See [Semantic Versioning (SemVer)](https://semver.org/) in respect of the terminology used in this document, specifically "Pre-Release Version Suffix".
 
-## Labs
+The Pre-Release `<build>` component should always be a positive integer value.
```

Comment:
Shouldn't this be under the `## Pre-Releases`? :wink:

Reply:
My logic was that I was grouping it with the previous para relating to SemVer, but your point is very valid. I shall move it. 👍

Reply:
https://github.com/ably/engineering/pull/20/commits/f11c97856714c5ca95ec79ebba339c5e23c140aa